patches/wp-lister-amazon.0001.feature.added-order-to-filter.patch

```diff
@@ -0,0 +1,28 @@
+From b8ccca0bb7876223009b2a9694a1606177d9eeb9 Mon Sep 17 00:00:00 2001
```

Comment:
A better name would be
```
feature.shipping-service-map-filter-order
```
Can you rename the file?

Reply:
Done.